### PR TITLE
Header and Footer

### PR DESCRIPTION
**Pull request by vineethvanga18**

### CODE REVIEW

File: frontend/src/components/Header.vue [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    div
+        v-app-bar(app clipped-right color="primary" )
+            v-app-bar-nav-icon.hidden-md-and-up(dark @click.stop="drawer = !drawer")
+            v-avatar(size="48px")
+                v-img(src="../assets/logo.jpeg")
+            v-toolbar-title.hidden-sm-and-down.white--text
+                span.title.ml-3.mr-5 Devlup Labs&nbsp;
+            v-spacer
+            div.hidden-sm-and-down.my-2
+              v-btn(v-for="link in links" :key="link.text" :to="{name:link.text}" rounded depressed color="primary" exact) {{link.text}}
```
Comment (Tanuj22) [inline frontend/src/components/Header.vue:11]: Are the links working?

File: frontend/src/components/Header.vue [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+              v-btn(rounded depressed color="primary" @click="() => this.$vuetify.goTo('#bottom')") {{contact.text}}
```
Comment (Tanuj22) [inline frontend/src/components/Header.vue]: Contacts will be separate page right @anshulahuja98 ? We do not need to got to footer I think?

File: frontend/src/components/Header.vue (hunk 1 above)
Comment (vineethvanga18) [reply frontend/src/components/Header.vue:11]: yep

File: frontend/src/components/Header.vue (hunk 2 above)
Comment (anshulahuja98) [reply frontend/src/components/Header.vue]: Yep

Comment (vineethvanga18) [reply frontend/src/components/Header.vue]: This way we can avoid requesting for another page

Comment (Tanuj22) [reply frontend/src/components/Header.vue]: @vineethvanga18 Add the contact button with the test on the buttons.

Comment (anshulahuja98) [reply frontend/src/components/Header.vue]: Change contacts option to Team

Comment (anshulahuja98) [reply frontend/src/components/Header.vue]: and don't redirect it to bottom

Comment (vineethvanga18) [reply frontend/src/components/Header.vue]: Should I remove contacts from footer

Comment (anshulahuja98) [reply frontend/src/components/Header.vue]: Footer probably just has the social links right?
Let them be there

Or did you include other stuff as well?

Comment (vineethvanga18) [reply frontend/src/components/Header.vue]: only social links

Comment (anshulahuja98) [reply frontend/src/components/Header.vue]: Let them be there then

Comment (vineethvanga18) [reply frontend/src/components/Header.vue]: So there are the changes needed?
Change contacts option to Team
and don't redirect it to bottom

Comment (vineethvanga18) [reply frontend/src/components/Header.vue]: *these

File: frontend/src/router/index.js [hunk 3]
```diff
@@ -15,6 +15,26 @@ const routes = [
...
+  },
+  {
+    path: "/about",
+    name: "About",
+    component: Home
+  },
+  {
+    path: "/events",
+    name: "Events",
+    componet: Profile
```
Comment (anshulahuja98) [inline frontend/src/router/index.js]: Typo

File: frontend/src/router/index.js [hunk 4]
```diff
@@ -15,6 +15,26 @@ const routes = [
...
+  },
+  {
+    path: "/timeline",
+    name: "Timeline",
+    componet: Home
```
Comment (anshulahuja98) [inline frontend/src/router/index.js]: Typo

File: frontend/src/router/index.js [hunk 5]
```diff
@@ -15,6 +15,26 @@ const routes = [
...
+  },
+  {
+    path: "/blog",
+    name: "Blog",
+    componet: Profile
```
Comment (anshulahuja98) [inline frontend/src/router/index.js]: Typo

**Review by anshulahuja98 — CHANGES_REQUESTED** [review]

File: frontend/src/router/index.js (hunk 3 above)
Comment (vineethvanga18) [reply frontend/src/router/index.js:27]: We dont have those components yet

File: frontend/src/components/Header.vue (hunk 2 above)
Comment (Tanuj22) [reply frontend/src/components/Header.vue]: Yes, you can include this button with the above button.

File: frontend/src/components/Header.vue [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+        v-navigation-drawer(v-model="drawer" app left temporary)
+          v-list(dense)
+            v-list-item( v-for="link in links" :key="link.text" @click="" :to="{name:link.text}" exact)
+              v-list-item-action
+                v-icon {{link.icon}}
+              v-list-item-content
+                v-list-item-title(class="grey--text") {{link.text}}
```
Comment (Tanuj22) [inline frontend/src/components/Header.vue]: Use pug for `class`

File: frontend/src/components/Header.vue [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+            v-list-item( @click="()=>{this.$vuetify.goTo('#bottom');drawer = false;}" ) 
+              v-list-item-action
+                v-icon {{contact.icon}}
+              v-list-item-content
+                v-list-item-title(class="grey--text") {{contact.text}}
```
Comment (Tanuj22) [inline frontend/src/components/Header.vue]: Use pug for `class`

**Review by Tanuj22 — CHANGES_REQUESTED** [review]

File: frontend/src/components/Footer.vue [hunk 8]
```diff
@@ -0,0 +1,18 @@
+<template lang="pug">
+  div(id="bottom")
```
Comment (Tanuj22) [inline frontend/src/components/Footer.vue]: No use of this `bottom` now

**Pull request merged**